Tests/AK/TestIPv6Address.cpp

```diff
@@ -13,9 +13,7 @@ TEST_CASE(should_default_contructor_with_0s)
 {
     constexpr IPv6Address addr {};
 
-    static_assert(addr.is_zero());
-
-    EXPECT(addr.is_zero());
+    EXPECT_AND_STATIC_ASSERT(addr.is_zero());
```

Comment:
As mentioned in the previous commit, the run time checks likely get folded away,
maybe look at what I did for Optional, testing key functionality in a consteval function